### PR TITLE
documented kyc folder

### PR DESCRIPTION
**Pull request by manddy**
Fixes #{Issue Number}
#183 

 Summary 
Changed the presenter and fragment files for all levels kyc

### CODE REVIEW

**Comment by manddy** [comment]
@ankurs287  kindly review this PR

**Comment by ankurs287** [comment]
@manddy follow the rules mentioned here: https://www.tutorialspoint.com/java/java_documentation.htm

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/KYCContract.java [hunk 1]
```diff
@@ -8,19 +8,49 @@
...
+ * @author ankur
+ * @since 11/July/2018
  */
 
 public interface KYCContract {
 
     interface KYCDescriptionView extends BaseView<KYCDescriptionPresenter> {
 
+        /**
+         * Defining onFetchLevelSuccess function in the
```
Comment (ankurs287) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/KYCContract.java]: use 'Declaring' not 'Defining'

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/KYCContract.java [hunk 2]
```diff
@@ -8,19 +8,49 @@
...
+         * KYCContract file  to check the  current kyc level achieved by the user
```
Comment (ankurs287) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/KYCContract.java]: KYC interface not file

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/KYCContract.java [hunk 3]
```diff
@@ -8,19 +8,49 @@
...
+         * and implementing it in the KYCDescription Presenter
```
Comment (ankurs287) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/KYCContract.java]: wrong. onFetchLevelSuccess is implemented in View class of KYC

Comment (ankurs287) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/KYCContract.java]: @manddy there are many issues with the documentation. Understand the flow of the Use case properly.

**Comment by luckyman20** [comment]
@manddy Are you still working on this, if not, please let us know so that we can assign the issue to someone else.

**Comment by luckyman20** [comment]
@manddy I am closing this PR because of your inactivity for a longer period of time so that we are not blocked and others can work on this issue. Thanks for your contribution.

**Pull request closed without merging**